src/lib.rs

```diff
@@ -281,7 +325,10 @@ pub struct PanCam {
 impl Default for PanCam {
     fn default() -> Self {
         Self {
+            #[cfg(not(feature = "leafwing-input-manager"))]
             grab_buttons: vec![MouseButton::Left, MouseButton::Right, MouseButton::Middle],
+            #[cfg(feature = "leafwing-input-manager")]
+            grab_buttons: vec![],
```

Comment:
Hmm... I'm not sure how I feel about changing the defaults based on a feature flag... could lead to subtle surprising behavior/bugs if you have multiple crates that depend on pancam?

Reply:
I didn't want to have the default be using standard input if one requested leafwing. I made a big change around this now (will detail it in the main thread).